scripts/workon.bat

```diff
@@ -35,8 +35,12 @@ if not "%1"=="" (
 
 if defined VIRTUAL_ENV (
     call "%VIRTUAL_ENV%\Scripts\deactivate.bat"
+    if exist "%VIRTUAL_ENV%\Scripts\postdeactivate.bat" (

```

Comment:
`deactivate.bat`  clears `%VIRTUAL_ENV%`, so this works by taking advantage of DOS's rather special variable expansion rules.  It might be clearer if the postdeactivate.bat path was saved before the `call ... deactivate.bat`?

Reply:
Good point, done